village/src/components/SmurfForm.js

```diff
@@ -1,25 +1,33 @@
-import React, { Component } from 'react';
+import React, { Component } from "react";
+import axios from "axios";
 
 class SmurfForm extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      name: '',
-      age: '',
-      height: ''
+      name: "",
+      age: "",
+      height: ""
     };
   }
 
   addSmurf = event => {
+    const [name, age, height] = [this.state.name, Number(this.state.age), this.state.height];
     event.preventDefault();
     // add code to create the smurf using the api
+    axios.post('http://localhost:3333/smurfs', { name, age, height })
+    .then(res => {
+      this.props.fetchSmurfs();
```

Comment:
You did not pass down `.fetchSmurfs` function through `props` to this component. So, line 20 is useless in this case.